static/src/stylesheets/module/content/_truncation.scss

```diff
@@ -11,19 +11,9 @@
         display: block;
         position: absolute;
         bottom: 0;
-        background: linear-gradient(top, rgba(255, 255, 255, 0), rgba(255, 255, 255, 1));
-        height: 200px;
         width: 100%;
-        border-bottom: solid 40px #ffffff;
-    }
-}
-
-.content__read-more-button {
-    display: none;
-
-    .content__article-body--truncated & {
-        display: block;
-        position: absolute;
-        bottom: 0;
+        height: 40px;
+        background-color: #ffffff;
+        box-shadow: 0 0 120px 60px #ffffff;
```

Comment:
this is a very expensive thing to render, it looks like a bit one too. bit nervous about it...